clarify why there is no GStreamer import support anymore (not some kind of build/install issue)

diff --git a/src/AboutDialog.cpp b/src/AboutDialog.cpp
index d6d2046101c6e7b487b9726c831368ea88c049e1..1e06fea347d1e877301fecb795021ebd99fd6054 100644
--- a/src/AboutDialog.cpp
+++ b/src/AboutDialog.cpp
@@ -528,7 +528,7 @@ void AboutDialog::PopulateInformationPage( ShuttleGui & S )
    #ifdef USE_GSTREAMER
    AddBuildinfoRow(&informationStr, wxT("gstreamer"), _("Import via GStreamer"), enabled);
    #else
-   AddBuildinfoRow(&informationStr, wxT("gstreamer"), _("Import via GStreamer"), disabled);
+   AddBuildinfoRow(&informationStr, wxT("gstreamer"), _("Import via GStreamer - removed in version 2.0.6"), disabled);
    #endif
 
    informationStr += wxT("</table>\n");  //end table of file format libraries
